elpy.el

```diff
@@ -3113,7 +3113,8 @@ Points to file FILE, at position POS."
 
   (defun elpy-xref--identifier-at-point ()
     "Return identifier at point."
-    (symbol-at-point))
+    (when (symbol-at-point)
+        (substring-no-properties (symbol-name (symbol-at-point)))))
```

Comment:
Considering `symbol-at-point` basically does `(thing-at-point 'symbol)`, which in turn returns a string, I would suggest using that directly.